ui/page/main_page.go

```diff
@@ -59,15 +60,20 @@ type MainPage struct {
 	usdExchangeSet  bool
 	totalBalance    dcrutil.Amount
 	totalBalanceUSD string
+
+	hideBalance decredmaterial.IconButton
+	hideValue   bool
+	hiddenValue bool
```

Comment:
```suggestion
	isBalanceHidden bool
```
Those two booleans are redundant and give and appropriate name to it